app/controllers/slack_controller.rb

```diff
@@ -0,0 +1,27 @@
+require "#{Rails.root}/lib/slack"
+
+class SlackController < ApplicationController
+  include Slack
```

Comment:
I think you don't need to include Slack here or the require at the top of the page, the `lib` directory should be autoloaded and be available straight away.